### DISCUSSION TITLE
google colab

### DISCUSSION THREAD

**Discussion by DidManconcept** [mashb1t/Fooocus#26] [Q&A] [answered]
hello
google colab
https://colab.research.google.com/github/mashb1t/Fooocus/blob/main/fooocus_colab.ipynb
ne fonctionne plus ???
urllib.error.HTTPError: HTTP Error 503: Service Temporarily Unavailable
dommage

**Comment by GinvKazane** [comment]
Hello
After launching the cell "Run" Google Collab displays at the end:

Traceback (most recent call last):
  File "/content/Fooocus/entry_with_update.py", line 46, in <module>
    from launch import *
  File "/content/Fooocus/launch.py", line 141, in <module>
    from webui import *
  File "/content/Fooocus/webui.py", line 10, in <module>
    import modules.async_worker as worker
  File "/content/Fooocus/modules/async_worker.py", line 50
    from modules.sdxl_styles import get_random_style, random_style_name, apply_style, apply_wildcards, fooocus_expansion, apply_arrays,
                                                                                                                                       ^
SyntaxError: trailing comma not allowed without surrounding parentheses

**Reply by mashb1t** [reply, marked as answer]
Solved a while ago, sorry for not answering. Please restart Fooocus / clone again, works now.

**Comment by mrblaze27** [comment]
hello,
i've an issue when run Fooocus non Google Colab, the error is for connection timeout.
<img width="349" alt="error" src="https://github.com/mashb1t/Fooocus/assets/50958270/37268fb1-4a50-4ec2-bcc3-543cca3f4734">

**Reply by mashb1t** [reply]
@mrblaze27 please find more information in the troubleshooting guide at https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md

**Reply by mrblaze27** [reply]
i checked already, sorry but i don't find any info to solve my issue

**Reply by mashb1t** [reply]
Please open an issue and provide required data, especially the full console log from startup to the error and your system specs.

**Reply by mrblaze27** [reply]
Sorry, it's my fault, on another pc works well
